src/features/profile/components/collection-container/scores/ingredients/index.tsx

```diff
@@ -0,0 +1,215 @@
+                  >
+                    <Paragraph>
+                      <Title level={4}>{ingredient.name}</Title>
+                    </Paragraph>
+                    <Image
+                      preview={false}
+                      style={{ padding: '4px' }}
+                      src={`${process.env.ASSETS_URL}${ingredient.image64}`}
+                      alt="ingredientimage"
+                    />
+                    <Divider />
+                    {songs.map((song) => {
+                      if (ingredient.songId === song.id) {
+                        return (
+                          <>
+                            <JacketDifficulty
+                              className={song.difficulty}
+                              src={`${process.env.ASSETS_URL}${song.image64}`}
+                            />
+                            <Paragraph>{song.name}</Paragraph>
+                          </>
+                        );
+                      }
+                    })}
+
+                    <Paragraph>
+                      {scores.map((score) => {
+                        if (ingredient.songId === score.songId) {
+                          return (
+                            <Text>
+                              <Text style={{ fontWeight: 'bold' }}>
+                                EX Score:{' '}
+                              </Text>
+                              [{score.value}]
+                            </Text>
+                          );
+                        }
+                      })}
+                    </Paragraph>
+
+                    <Paragraph>
+                      {scores.map((score) => {
+                        if (ingredient.songId === score.songId) {
+                          return currentScores.map((cs) => {
+                            if (score.id === cs.scoreId) {
+                              return grades
+                                .filter((grade) =>
+                                  grade.id.includes(cs.gradedIngredientId),
+                                )
+                                .map((filteredName) => {
+                                  return (
+                                    <Rate
+                                      style={{ width: '100%' }}
+                                      disabled
+                                      defaultValue={gradeToInt(
+                                        filteredName.grade,
+                                      )}
+                                    />
+                                  );
+                                });
+                            }
+                          });
+                        }
+                      })}
+                    </Paragraph>
+                  </Card>
+                </IngredientsContainer>
+              </Space>
+            );
+          })}
+        </IngredientsContainer>
+      )}
+    </>
+  );
+};
+
+export default Ingredients;
```

Comment:
Are these cards the same as the ones that display on the ingredients page? Or different.

Reply:
If they are the same, we should be reusing the same component.

Reply:
I don't think we have a ingredients page :o .